wiki/Glossary/en.md

```diff
@@ -620,30 +621,31 @@ Pass has two meanings:
 
```

Comment:
`This topic was discussed in an episode of [osu!talk](/wiki/osu!talk) ([osu!talk special Over Mapping](https://www.youtube.com/watch?v=RepSYE3hN3A)) that overmapping actually has a positive effect in [osu!taiko](/wiki/osu!taiko) beatmaps because the drums plays an important role in a song.`

```
- The completion of a [beatmap](/wiki/beatmap) with or without the use of mods.
- The player having more than 50% health during a [break](#break).
  - An indicator will display if you passed the section.
```

Reply:
for lines 619 to 621
```
- The completion of a [beatmap](/wiki/Beatmaps) with or without the use of mods.
- The player having more than 50% health during a [break](#break).
  - An indicator will display if you passed the section.
```